components/mpas-albany-landice/cime_config/buildnml

```diff
@@ -70,6 +71,7 @@ def buildnml(case, caseroot, compname):
     elif glc_grid == 'mpas.ais8to30km':
         grid_date += '20221027'
         grid_prefix += 'ais_8to30km'
+        datamode_date += '20250121'
```

Comment:
We will presumably add a similar file for `mpas.ais4to20km` in a follow-up PR?  I think that's the one that would be compatible with the SORRMr4 mesh I'm working on and the SORRMr5 mesh that would include a dry region.  But we could also make an Icos r8 (?) mesh that could be compatible with this lower res MALI mesh.  Forgive me if we've already discussed this and I've lost track.

Reply:
Yes, my plan was to add a datamode file for the recently updated 4km AIS mesh, most likely in a follow up PR.